package.json

```diff
@@ -8,7 +8,7 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
-    "start": "npm run develop && npm:api",
+    "start": "npm run develop",
```

Comment:
It would be nice though to start the API automatically:
`npm run develop & npm run api`

Reply:
That is very nice.